lib/ts/recipe/session/with-jwt/recipeImplementation.ts

```diff
@@ -18,6 +18,7 @@ import { RecipeInterface } from "../";
 import { NormalisedAppinfo } from "../../../types";
```

Comment:
Extract logic of deleting "our" claims, creating a JWT and then updating the access token payload into a helper function.

Reply:
Done